### PR TITLE
logging

### PR DESCRIPTION
**Pull request by LeoTomatsu**
from issue #36 

- to add logging features, I used the oslo_log module provided by OpenStack. I took the print statements within `addusers.py` and `set-quotas.py` and replaced to oslo_log statements.

### CODE REVIEW

File: addusers.py [hunk 1]
```diff
@@ -312,9 +312,9 @@ def parse_rows(rows, select_user=None):
                 except:
                     # If the user typed something in this box but didn't
                     # follow instructions
-                    print ("WARNING: cannot add additional users to "
-                           "project `{}` from input: `{}`").format(
-                               entry[15], entry[17])
+                    LOG.warning(("WARNING: cannot add additional users to "
```
Comment (jeremyfreudberg) [inline addusers.py]: redundant word warning

File: addusers.py [hunk 2]
```diff
@@ -430,14 +430,16 @@ def parse_rows(rows, select_user=None):
                         raise ItemNotFoundError('User', user.name)
                     else:
                         # We don't treat this as a critical error
-                        print ("WARNING: Additional user `{}` does not exist "
-                               "in Keystone. The user will not be added to "
-                               "project {}").format(user.name,
-                                                    ks_project.name)
+                        LOG.warning(("WARNING: Additional user `{}` does "
```
Comment (jeremyfreudberg) [inline addusers.py]: ditto

File: addusers.py (hunk 1 above)
Comment (ltomatsu) [inline addusers.py]: got it!

File: addusers.py (hunk 2 above)
Comment (ltomatsu) [inline addusers.py]: haha yep I'll get rid of those

**Comment by jeremyfreudberg** [comment]
superseded by #86

**Pull request closed without merging**